public/components/FeatureAnywhereContextMenu/AnywhereParentFlyout/AnywhereParentFlyout.tsx

```diff
@@ -17,7 +17,7 @@ const AnywhereParentFlyout = ({ startingFlyout, ...props }) => {
   const [selectedDetectorId, setSelectedDetectorId] = useState();
 
   const AnywhereFlyout = {
-    create: AddAnomalyDetector,
```

Comment:
we shouldn't remove this line, instead, please append the `associated: AssociatedDetectors,` as the second line here